### PR TITLE
Updating Enterprise component versions for v3.5.0 release.

### PR DESCRIPTION
**Pull request by stevegaossou**
## Description
- Pin enterprise versions to v3.5.0
- Update repo path

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

File: pkg/components/images.go
Comment (bcreane) [inline pkg/components/images.go:1]: 2020-2021?

File: config/enterprise_versions.yml
Comment (stevegaossou) [inline config/enterprise_versions.yml:75]: This comment block is old and no longer relevant.

**Review by doublek — APPROVED** [review]
✅

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
